onadata/apps/api/tests/viewsets/test_project_viewset.py

```diff
@@ -1946,6 +1947,9 @@ def test_last_date_modified_changes_when_adding_new_form(self):
 
         self._make_submissions()
 
+        # run cronjob to update date_modified field
+        apply_project_date_modified_async.delay()
```

Comment:
@KipSigei Did we make any changes to how a `Project.date_modified` is updated when a form is published?

Reply:
@kelvin-muchiri On the line above when a submission is made, with the changes introduced in this PR the `date_modified` field won't change until the cronjob is run. This hasn't changed how the date_modified field is updated when publishing a new form